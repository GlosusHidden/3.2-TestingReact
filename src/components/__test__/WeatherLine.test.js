import React from 'react'
import renderer from 'react-test-renderer'

import WeatherLine from '../WeatherLine';

it('WeatherLine Component Snapshot', () => {
  const renderedValue = renderer.create(
    <WeatherLine  param='Weather' value='Sun' />
  ).toJSON();
  expect(renderedValue).toMatchSnapshot();
});
