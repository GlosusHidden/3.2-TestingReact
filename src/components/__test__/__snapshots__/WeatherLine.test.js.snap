// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WeatherLine Component Snapshot 1`] = `
<div
  className="line"
>
  <div
    className="param"
  >
    Weather
  </div>
  <div
    className="value"
  >
    Sun
  </div>
</div>
`;
